src/less/date-textbox/date-textbox.less

```diff
@@ -0,0 +1,29 @@
+@import "../variables/variables.less";
+@import "../mixins/private/token-mixins.less";
+
+.date-textbox {
+    // 343px - (2 * @spacing-200) [not autmatically compiled by LESS]
+    --calendar-month-width: 311px;
+}
+
+.date-textbox__popover {
+    .background-color-token(calendar-background-color, color-background-primary);
+    border-radius: @spacing-200;
+    box-shadow: 0 2px 7px rgba(0, 0, 0, 0.15), 0 5px 17px rgba(0, 0, 0, 0.2);
```

Comment:
This is defined as `--bubble-filter` variable in bubble-mixins 
Maybe we need to move this to tokens .

https://github.com/eBay/skin/blob/master/src/less/mixins/private/bubble-mixins.less#L4-L5
Regardless, I'd try to use as much as you can from that file.

Reply:
There's a huge difference between the light, which looks good and the dark, which looks a bit broken.
<img width="887" alt="image" src="https://user-images.githubusercontent.com/1675667/222211461-a40a764f-7ba1-4b9d-98bd-3a7b77b2e91e.png">

Reply:
Figma doesn't have a dark mode example, but I imagine that they want something like what they have in the [dropdown menu popover](https://www.figma.com/file/zEBdEhbonrBOGzZ0fXzWvM/eBay-Design-System?node-id=31948:34280&t=23Ht9L6hvygXKAY7-0). This will be addressed in the follow-up PR (see above).